utils/evaluate_utils.py

```diff
@@ -92,9 +93,50 @@ def class_PR_curves(
     return fig, PR_data
 
 
+def model_cm(
+    log_reg_model: LogisticRegression, dataset: pd.DataFrame
+) -> pd.DataFrame:
+    """
+    display confusion matrix for logistic regression model on dataset
+
+    Parameters
+    ----------
+    log_reg_model : LogisticRegression
+        logistic regression model to evaluate
+    dataset : pd.DataFrame
+        dataset to evaluate model on
+
+    Returns
+    -------
+    pd.DataFrame
+        confusion matrix of model evaluated on dataset
+    """
+
+    # get features and labels dataframes
+    X, y = get_X_y_data(dataset)
+
+    # get predictions from model
+    y_pred = log_reg_model.predict(X)
+
+    # create confusion matrix
+    conf_mat = confusion_matrix(y, y_pred, labels=log_reg_model.classes_)
+    conf_mat = pd.DataFrame(conf_mat)
+    conf_mat.columns = log_reg_model.classes_
+    conf_mat.index = log_reg_model.classes_
```

Comment:
Consider making use of `pd.DataFrame`'s built-in parameters to specify the columns and index on instantiation.

Reply:
Thank you!

Reply:
Added in https://github.com/WayScience/phenotypic_profiling_model/pull/8/commits/9ebc7453f0667f15c00f929e9f15016a415e8b77